kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -104,6 +104,12 @@ func NewJobSubmitCommand(streams genericclioptions.IOStreams) *cobra.Command {
 		Example: jobSubmitExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			entryPointStart := cmd.ArgsLenAtDash()
+			if entryPointStart == -1 {
```

Comment:
if entryPointStart == -1 || len(args[entryPointStart:]) == 0 {

Reply:
Can cmd.ArgsLenAtDash return -1? If so what does it mean?

Reply:
cmd.ArgsLenAtDash returning -1 just means that there isn't a "--"